scripts/swebench_rna_one.py

```
@@ -0,0 +1,1462 @@
+    env: Mapping[str, str],
+    stdout_path: Path,
+    stderr_path: Path,
+    timed_trace_path: Path,
+    timeout: float | None,
+    monitor: FirstEditMonitor,
+) -> CommandResult:
+    stdout_path.parent.mkdir(parents=True, exist_ok=True)
+    started_wall = utc_now()
+    started = time.monotonic()
+    monitor.start()
+    process = subprocess.Popen(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)